shardingsphere-governance/shardingsphere-governance-core/shardingsphere-governance-core-facade/src/main/java/org/apache/shardingsphere/governance/core/facade/GovernanceFacade.java

```diff
@@ -100,6 +103,7 @@ public void onlineInstance() {
     @Override
     public void close() {
         repositoryFacade.close();
+        log.info("GovernanceRepositoryFacade has been closed.");
```

Comment:
Any reason to add this log?
UNIX philosophy is "No news is good news", I prefer do not add unnecessary logs